test/kuttl/tests/manila-basic/04-assert.yaml

```diff
@@ -0,0 +1,193 @@
+        reason: Ready
+        status: "True"
+        type: DBSyncReady
+      - message: Input data complete
+        reason: Ready
+        status: "True"
+        type: InputReady
+      - message: Setup complete
+        reason: Ready
+        status: "True"
+        type: ManilaAPIReady
+      - message: Setup complete
+        reason: Ready
+        status: "True"
+        type: ManilaSchedulerReady
+      - message: Deployment completed
+        reason: Ready
+        status: "True"
+        type: ManilaShareReady
+      - message: MariaDBAccount creation complete
+        reason: Ready
+        status: "True"
+        type: MariaDBAccountReady
+      - message: " Memcached instance has been provisioned"
+        reason: Ready
+        status: "True"
+        type: MemcachedReady
+      - message: NetworkAttachments completed
+        reason: Ready
+        status: "True"
+        type: NetworkAttachmentsReady
+      - message: RabbitMqTransportURL successfully created
+        reason: Ready
+        status: "True"
+        type: RabbitMqTransportURLReady
+      - message: RoleBinding created
+        reason: Ready
+        status: "True"
+        type: RoleBindingReady
+      - message: Role created
+        reason: Ready
+        status: "True"
+        type: RoleReady
+      - message: ServiceAccount created
+        reason: Ready
+        status: "True"
+        type: ServiceAccountReady
+      - message: Service config create completed
+        reason: Ready
+        status: "True"
+        type: ServiceConfigReady
+   databaseHostname: openstack.manila-kuttl-tests.svc
+   manilaAPIReadyCount: 1
+   manilaSchedulerReadyCount: 3
+   manilaSharesReadyCounts:
+      share0: 1
+   transportURLSecret: rabbitmq-transport-url-manila-manila-transport
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: manila-db-purge
+spec:
+  jobTemplate:
+    metadata:
+      labels:
+        service: manila
+    spec:
+      completions: 1
+      parallelism: 1
+      template:
+        spec:
+          containers:
+          - args:
+            - -c
+            - /usr/bin/manila-manage --config-dir /etc/manila/manila.conf.d
+              db purge 30
+            command:
+            - /bin/bash
+            name: manila-db-purge
+            volumeMounts:
+            - mountPath: /etc/manila/manila.conf.d
+              name: db-purge-config-data
+              readOnly: true
+          serviceAccount: manila-manila
+          serviceAccountName: manila-manila
+          volumes:
+          - name: db-purge-config-data
+            secret:
+              defaultMode: 420
+              items:
+              - key: 00-config.conf
+                path: 00-config.conf
+              secretName: manila-config-data
+  schedule: 1 0 * * *
+  suspend: false
+---
+# when using image digests the containerImage URLs are SHA's so we verify them with a script
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+commands:
+  - script: |
```

Comment:
please add `set -euxo pipefail` to avoid false positives.

Reply:
All set, ty.